lib/attachinary/utils.rb

```diff
@@ -79,5 +104,20 @@ def self.process_options(options)
       options
     end
 
+    def self.extract_photo_attributes(hash)
+      attributes = hash.slice(*ALLOWED_PHOTO_ATTRIBUTES)
+      attributes[TRANSFORMATION_KEY] = extract_transformation_attributes(hash) if hash.has_key?(TRANSFORMATION_KEY)
+      attributes
+    end
+
+    def self.extract_transformation_attributes(hash)
```

Comment:
Yeah, this is the kind of coercion that makes me want to take a slightly different approach like we talked about yesterday.